lib/config.js

```diff
@@ -4,11 +4,11 @@ const maxFileRaw = process.env.MAX_FILESIZE_MB;
 const maxFileParsed = Number(maxFileRaw);
 
 // eslint-disable-next-line eqeqeq
-const appPort = (appPortRaw == appPortParsed) ? appPortParsed : 4000;
+const appPort = (appPortRaw === appPortParsed) ? appPortParsed : 4000;
```

Comment:
This is an intentional double equals comparison.

Basically we want to check here if `("4000" == 4000)` so this needs to be a loose check on the type, otherwise the app will always default to port 4000 without ability to override that.